code/orphaned_procs/priority_announce.dm

```diff
@@ -33,12 +33,9 @@
 /proc/print_command_report(text = "", title = "Central Command Update")
 	for (var/obj/machinery/computer/communications/C in machines)
 		if(!(C.stat & (BROKEN|NOPOWER)) && C.z == ZLEVEL_STATION)
-			var/obj/item/weapon/paper/P = new /obj/item/weapon/paper( C.loc )
-			P.name = "paper- '[title]'"
-			P.info = text
 			C.messagetitle.Add("[title]")
 			C.messagetext.Add(text)
-			P.update_icon()
+			C.new_printjob("paper- '[title]'", text)
```

Comment:
I need to fix this part since it loops, waits for the print job to finish, and rinse repeat until all computers have printed the report.